spec/Cabin/Cabin.vspec

```diff
@@ -130,33 +131,27 @@ DoorCount:
 # Row1 is the frontmost row.
 # Row2-RowX are subsequent rows of seats.
 #
-# Within each row there are up to Y seat positions
-# Pos1 is leftmost.
-# PosY is rightmost.
-#
-# The value of Y shall consider the row with most number of seats in the vehicle.
-# If a vehicle e.g. have 2 seats in front row and 3 seats in second row, then Pos
-# shall be defined as Pos[1,3]
 # Which seats that actually exist can be defined by SeatPosCount
 ##
 
 Seat:
   type: branch
   instances:
     - Row[1,2]
-    - Pos[1,3]
+    - ["Driver","Middle","Passenger"]
   description: All seats.
 #include SingleSeat.vspec Seat
 
 #
 # Seat attributes.
 #
 DriverPosition:
-  datatype: uint8
```

Comment:
This makes a lot of sense, but wouldn't it then be better not touching the seats/doors because while it is useful for the front row, it seems a little weird for everything else.

Also, may we want to add "center" to allowed (basically all Vehicles having only one seat, Tesla Semi, Forklifts, racecars).

Reply:
Or "middle" as used for Seat above, as long as we use same term (middle or center) I am happy.

But if we support middle/center for driver position, then we need to discuss how to handle other signals that possibly use Driver/Passenger as instances, like doors, mirrors, (rear) seats and so on. I can see a few options:

- Consider those vehicles to not have "driver-side", and require them to use an overlay to redefine instances just as we might do for motorbikes.
- Add another signal/attribute `Vehicle.LogicalDriverPosition` that just contains Left and Right which can be used to identify for example which mirror which is considered to be the "driver mirror" and which is the "passenger mirror". (I was thinking first about a signal indicating if a vehicle is intended for left side traffic or right side traffic, and deduce logical driver side from that, but then I realized that there sometimes are corner cases like postal-vehicles where the driver sits on the "wrong side" for convenience reasons)